api/install/service.go

```diff
@@ -9,23 +9,20 @@ import (
 	"github.com/gojekfarm/albatross/pkg/helmcli/flags"
 )
 
-// TODO: Move the service interface to a common place for all apis
-type service interface {
-	Install(ctx context.Context, req Request) (Response, error)
+type Service struct {
+	cli helmcli.Client
```

Comment:
we could've had just interface installer as dependency instead of fat client interface